mlrun/db/filedb.py

```diff
@@ -41,22 +46,19 @@ def connect(self, secrets=None):
         return self
 
     def store_log(self, uid, project='', body=None, append=True):
-        filepath = self._filepath('runs', project, uid, '') + '.log'
+        filepath = self._filepath(runs_dir, project, uid, '') + '.log'
         # TODO: handle append
```

Comment:
@tebeka lets make it a constant value, better name it run_logs

Reply:
Done